### ISSUE TITLE
Catastrophic forgetting when trying to fine-tune OS-ATLAS-Base-7B on grounding

### ISSUE THREAD

**Issue by PranitChawla** [OS-Copilot/OS-Atlas#36]
Hi, 

Thanks for the great work. While I have been able to get inference on the pre-trained OS-ATLAS-Base-7B to work, when I try to fine-tune the model using LLaMA-factory using various settings (frozen ViT layer, lower learning rate, etc.) on additional custom grounding data following the exact prompt and data format mentioned in the inference code, I observe that the performance goes down very significantly on grounding benchmarks like Screenspot even though the train loss goes down. Do you know why this might be happening?

Thank you.
